File: app/src/main/java/org/thoughtcrime/securesms/components/webrtc/WebRtcLocalRenderState.java
package org.thoughtcrime.securesms.components.webrtc;

public enum WebRtcLocalRenderState {
  GONE,
  SMALL_RECTANGLE,
  SMALLER_RECTANGLE,
  LARGE,
  LARGE_NO_VIDEO,
  EXPANDED;

  public boolean isAnySmall() {
    return this == SMALL_RECTANGLE || this == SMALLER_RECTANGLE;
  }
}

File: app/src/main/java/org/thoughtcrime/securesms/components/webrtc/WebRtcLocalRenderState.kt
package org.thoughtcrime.securesms.components.webrtc

enum class WebRtcLocalRenderState {
  GONE,
  SMALL_RECTANGLE,
  SMALLER_RECTANGLE,
  LARGE,
  LARGE_NO_VIDEO,
  EXPANDED,
  FOCUSED;

  val isAnySmall: Boolean
    get() = this == SMALL_RECTANGLE || this == SMALLER_RECTANGLE
}
